docs/manage/CLI.md

```diff
@@ -498,7 +498,7 @@ The `meta.json` file includes the following configuration options:
     <td><code>entrypoint</code></td>
     <td>string</td>
     <td><strong>Required</strong></td>
-    <td>The name of the file that starts your module program. This can be a compiled executable, a script, or an invocation of another program.</td>
+    <td>The name of the file that starts your module program. This can be a compiled executable, a script, or an invocation of another program. If you are providing your module as a single file to the <code>upload</code> command, provide the path to that single file. If you are providing a directory containing your module to the <code>upload</code> command, provide the path to the entry point file contained within that directory.</td>
```

Comment:
Gotcha. I included this explicit callout in a few places because we have new ways to fail validation / usage in the registry:

- If I provide a single file to `upload` as `<module-path>` but specify anything other than that single file to `entrypoint` (I must supply different values for all other `upload` types, archive and directory, for example, so this becomes a helpful differentiation from previous process, for example).
- If I provide a directory to `upload` as `<module-path>` but specify a path or file outside of that directory to `entrypoint`.

Or: let me know if I misinterpreted the change!!